docs/github.md

```diff
@@ -0,0 +1,41 @@
+# Using Jujutsu with Github Projects
+
+## Setup a ssh key
+As of December 2022 it's necessary to setup a ssh key to work with github
+projects. See [Github's Tutorial][gh]. This restriction may be lifted in the 
+future, see issue #469 for more information and progress on authenticated http. 
```

Comment:
GitHub markdown doesn't seem to auto-link the "#469", so I guess we'll have to make it

Reply:
Done